compose/completion.md

```diff
@@ -4,6 +4,8 @@ keywords: fig, composition, compose, docker, orchestration, cli, reference
 title: Command-line completion
 ---
 
+{% assign composeversion = '1.15.0' %}
```

Comment:
Is there a way to define variables upstream? This variable is also defined in compose/install.md

Reply:
@albers I can add a variable to these files on your PR